Update `SorobanRpc` to `rpc` and use `StellarRpc`

jeesunikim opened the pull request:
1. js-stellar-sdk [v13.0.0](https://github.com/stellar/js-stellar-sdk/releases/tag/v13.0.0) removed `SorobanRpc`. 

> The SorobanRpc import, previously deprecated, has been removed. You can import rpc instead:

2. We rebranded `SorobanRpc` to `StellarRpc` in November 2024

This PR is to update the doc accordingly.

stellar-jenkins commented:
**Preview is available here:**<br>http://developers-pr1298.previews.kube001.services.stellar-ops.com

Shaptic reviewed (approved):
I don't get why you'd do `rpc as StellarRpc` instead of just using `rpc`, but I don't have an opinion either way :) LGTM!

jeesunikim commented:
@Shaptic so i didn't wanna change the entire existing `SorobanRpc` to `rpc` so I started as only to update `rpc as SorobanRpc` and leave the rest as it is then I realized we renamed `SorobanRpc` to `StellarRpc` so I updated that then I realized 'oh I could've just used rpc as it is..' okay maybe i will just use rpc

Shaptic commented:
@jeesunikim either way works for me! But to be clear we didn't rename `SorobanRpc` -> `StellarRpc` _in the SDKs_, just as a "product name". In the SDK it's only `rpc`.

jeesunikim commented:
yep i am aware!
